### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by petudeveloper**
In the third milestone,  you will complete the last sections of the website. 

    - In the magazine and Editor’s pick
    - Newspaper main sections
    - Subscribe / Newsletter

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+#input-sear-top-nav {
+  width: 28%;
+  outline: none;
+}
+
+.hover-red:hover {
+  color: #f72210 !important;
+}
```
Comment (SajjadAhmad14) [inline styles/main.css]: Please do not use !important rule so often.  Read <a href = 'https://speckyboy.com/good-bad-css-practices/'>this article</a> for further reference.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello Mario Rodriguez Cota, and David Alvarez Mazzo! You have been doing a tremendous job so far :ok_hand:
## But there are some changes to make in order to get your project approved.

- [x] Your page has a horizontal scroll, please fix that.
- [x] Please use Semantic HTML tags because with them it's easier for both people and machines to read and understand it. You should use a section tag for each section and a footer tag for your footer.
- [x]  Please do not use !important rule.
### Subscribe / Newsletter
- [x] Newsletter section would have an input with type email, but you have input type text.
- [x] Please fix this navbar for desktop viewport.
nav-yours

## Happy Coding! 

### Please feel free to contact me on slack (@Sajjad Ahmad).

**Comment by petudeveloper** [comment]
@SajjadAhmad14 thank you for your comments and suggestions. They helped us to clean the code and see some missing things we had.

We correct all your suggestions:
- Remove horizontal scroll.
- Use Semantic HTML tags in the footer and sections needed. 
- Remove !important rule in CSS file.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @mariordgez @petumazo, good job on this milestone! 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Well done team


[ OPTIONAL ]

- You have a lot of empty spaces between your code, erase the empty spaces, in that way your code will be super handsome and professional.

Now you can merge this branch 🚀 

You can contact me on slack if you have any questions, just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

**Comment by petudeveloper** [comment]
@mricanho thanks for your suggestion. We did the change you suggested:

- Delete empty spaces in the code.

**Pull request merged**